Add sway support

rkubosz opened the pull request:
Hi! I created a base16 template for sway and I integrated it with base16-manager. This PR contains a function, that switches themes for sway. I did not write a code to nest base16-sway bindings in user's sway config. IMO it would be better to not touch user's config. User can do it by himself, as described in [readme of base16-sway](http://github.com/rkubosz/base16-sway), especially that this operation should be done only once.

rkubosz committed:
add support for base16-sway

rkubosz committed:
update readme

Added section describing integration of base16-manager and sway

rkubosz committed:
fix issue suggested by shellcheck

AuditeMarlow commented:
Hey there, thanks for your contribution!

This looks like a fairly straightforward way to set a theme in Sway. I personally also prefer not touching existing user configs wherever possible, so kudos from me. :)

theova reviewed (commented):
Hello & thanks for your PR! 

Would it be possible to set sway by using the more general function `set_by_copy()`?

Personally, I can't test your code, but it seems ok. Altough it is quite simple, comments would help for a fast understanding :smile:

rkubosz committed:
change method for colorscheme applying for sway

Previously there was a custom function, that created a symlink to
colorscheme. Now it uses a previously defined function set_by_copy.

rkubosz commented:
As suggested, I changed the method to `set_by_copy()` and deleted my previous function `set_sway()`. It seems to work fine.

theova reviewed (approved):
Nice! Looks good to me :-)
